### PR TITLE
updated doc for v2 alerts

### PR DESCRIPTION
**Pull request by hitarth-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 1]
```diff
@@ -102,15 +105,13 @@ To configure a Microsoft Graph Security API Source:
     ![Mircrosoft Graph Security API Source input.png](/img/send-data/Microsoft-Graph-Security-API-Source-input.png)
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:105]: <br/>Microsoft-Graph-Security-API-Source-inputt

Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:105]: <br/>Microsoft-Graph-Security-API-Source-inputt

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 2]
```diff
@@ -102,15 +105,13 @@ To configure a Microsoft Graph Security API Source:
...
-1. **Forward to SIEM**. Check the checkbox to forward your data to Cloud SIEM Enterprise. If you click the checkbox, another option appears: Use **Dynamic Vendor and Product SIEM Metadata**.
-   *  If you don't checkmark **Use Dynamic Vendor and Product SIEM Metadata**, the metadata fields that identify vendor and product (`_siemVendor` and `_siemProduct`) will be set to _Security Graph API_.* If you do checkmark **Use Dynamic Vendor and Product SIEM Metadata**, CSE will retain the original product information. This is helpful when multiple data sources from multiple vendors and products are proxied through the Microsoft Security Graph API and you want the original vendor and product metadata information reflected in CSE. The table below shows the metadata fields that CSE sets.
-   <br/>
+1. **Forward to SIEM**. Check the checkbox to forward your data to Cloud SIEM Enterprise. When configured with the **Forward to SIEM** option the following metadata fields are set:
 
    | Field Name | Value   |
    | :-- | :-- | :-- |
    | _siemForward  | true |
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:112]:
```suggestion
   | _siemForward  | True |  
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 3]
```diff
@@ -75,14 +75,17 @@ The following steps show you how to create a service application:
...
-    You need to find and select the **SecurityEvents.Read.All** permission. See [this list](https://docs.microsoft.com/en-us/graph/permissions-reference#security-permissions) of all the available security permissions.
+    You need to find and select the **SecurityAlert.Read.All**, **SecurityAlert.ReadWrite.All**, **SecurityIncident.Read.All**, **SecurityIncident.ReadWrite.All** and **SecurityEvents.Read.All** permissions. See [this list](https://docs.microsoft.com/en-us/graph/permissions-reference#security-permissions) of all the available security permissions.
 
     :::note
     An Administrator must approve (grant) these permissions before the Source will function.
     :::
 
     ![6_ms_graph_app_add_permissions.png](/img/send-data/6_ms_graph_app_add_permissions.png)
 
+1. Also, an Application permission role is required. Click on **App roles**, then **Create app role**. Under this provide a display name for the role. Select **Applications** as the allowed member type. Provide **SecurityEvents.Read.All** as the value and provide a proper description for the same. Select the checkbox to enable the role and click **Apply**.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:86]: 1. Follow the steps below to enable **Application** permission role. 
	1. In the **Manage** menu select **App roles**.
	1. Click the **Create app role** tab.
		1. **Display name**. Enter the display name for the role. 
		1. **Allowed member types**. Select **Applications** as the allowed member type. 
		1. **Value**. Enter **SecurityEvents.Read.All** as the value.
		1. **Description**. Enter a brief description. 
		1. Select the checkbox to enable the app role .
	1. Click **Apply**.

**Review by JV0812 — CHANGES_REQUESTED** [review]
formatting

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 4]
```diff
@@ -51,38 +51,49 @@ The following steps show you how to create a service application:
...
 
-    ![5_ms_graph_app_client_secret_created.png](/img/send-data/5_ms_graph_app_client_secret_created.png)
+    <img src={useBaseUrl('/img/send-data/5_ms_graph_app_client_secret_created.png')} alt="5_ms_graph_app_client_secret_created" width="800"style={{border: '1px solid black'}} />
 
 1. Request the appropriate [permissions for the application](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-configure-app-access-web-apis#application-permission-to-microsoft-graph). Click on **API Permissions**, then **Add a permission** and select **Microsoft Graph**.
 
-    You need to find and select the **SecurityEvents.Read.All** permission. See [this list](https://docs.microsoft.com/en-us/graph/permissions-reference#security-permissions) of all the available security permissions.
+    You need to find and select the **SecurityAlert.Read.All**, **SecurityAlert.ReadWrite.All**, **SecurityIncident.Read.All**, **SecurityIncident.ReadWrite.All** and **SecurityEvents.Read.All** permissions. See [this list](https://docs.microsoft.com/en-us/graph/permissions-reference#security-permissions) of all the available security permissions.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:78]:
```suggestion
    You need to find and select the **SecurityAlert.Read.All**, **SecurityAlert.ReadWrite.All**, **SecurityIncident.Read.All**, **SecurityIncident.ReadWrite.All** and **SecurityEvents.Read.All** permissions. See [this list](https://docs.microsoft.com/en-us/graph/permissions-reference#security-permissions) to view all available security permissions.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 5]
```diff
@@ -51,38 +51,49 @@ The following steps show you how to create a service application:
...
 
+1. Follow the steps below to enable Application permission role.
+    - In the Manage menu select App roles.
+    - Click the Create app role tab.
+      - Display name. Enter the display name for the role.
+      - Allowed member types. Select Applications as the allowed member type.
+      - Value. Enter SecurityEvents.Read.All as the value.
+      - Description. Enter a brief description.
+      - Select the checkbox to enable the app role .
+    - Click Apply.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:94]:
```suggestion
1. Follow the steps below to enable the Application permission role.
   - In the Manage menu, select App roles.
   - Click the **Create app role** tab.
      - **Display name**. Enter the display name for the role.
      - **Allowed member types**. Select Applications as the allowed member type.
      - **Value**. Enter `SecurityEvents.Read.All` as the value.
      - **Description**. Enter a brief description.
      - Select the checkbox to enable the app role.
    - Click **Apply**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 6]
```diff
@@ -99,18 +110,16 @@ To configure a Microsoft Graph Security API Source:
...
-   <br/>
+1. **Forward to SIEM**. Check the checkbox to forward your data to Cloud SIEM Enterprise. When configured with the **Forward to SIEM** option the following metadata fields are set:
 
    | Field Name | Value   |
-   | :-- | :-- | :-- |
-   | _siemForward  | true |  
-   | _siemVendor  | This field is dynamically set based on the value of the vendor information in the log. <br/> `MS Sec Graph API {{vendorInformation.vendor}}`  |
-   | _siemProduct | This field is dynamically set based on the value of the vendor information in the log. <br/> `MS Sec Graph API{{vendorInformation.provider}}` |  
+   | :-- | :-- |
+   | _siemForward  | True |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:120]: @hitarth-sumo @JV0812 this should be lowercase as it's a boolean field

```suggestion
   | _siemForward  | true |  
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 7]
```diff
@@ -122,10 +131,11 @@ To configure a Microsoft Graph Security API Source:
 
 3. Enter the **Directory (tenant) ID**, **Application (client) ID**, and **Application Client Secret Value** you got from the Application you created in the [prerequisite](#prerequisite) step.
 
-4. The **Polling Interval** is set to 300 seconds by default, you can adjust it based on your needs.
-5. **Use Dynamic Vendor and Product SIEM Metadata**. Check the checkbox to dynamically set the vendor and product metadata information  (`_siemVendor` and `_siemProduct`) so that CSE retains the original product information. This is helpful when multiple data sources from multiple vendors and products are proxied through the Microsoft Security Graph API and you want the original vendor and product metadata information reflected in CSE.
+4. The **Polling Interval** is set to 5 minutes by default, you can adjust it based on your needs.
+
+5. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in Create a Processing Rule.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:136]:
```suggestion
4. The **Polling Interval** is set to 5 minutes by default. You can adjust it based on your needs.
5. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md [hunk 8]
```diff
@@ -122,10 +131,11 @@ To configure a Microsoft Graph Security API Source:
...
 
-6. When you are finished configuring the Source, click **Submit**.
+5. When you are finished configuring the Source, click **Submit**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:138]:
```suggestion
1. When you are finished configuring the Source, click **Submit**.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting/style

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md (hunk 2 above)
Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md:121]: as this is a boolean value, it should stay lowercased

**Review by kimsauce — APPROVED** [review]

**Review by JV0812 — APPROVED** [review]

**Pull request merged**